app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepository.kt

```diff
@@ -308,6 +330,15 @@ object AppRepository {
         updateSessions(sessions)
     }
 
+    /**
+     * Loads the session which has been selected at last.
+     */
+    @WorkerThread
+    fun loadSelectedSession(): Session {
+        val sessionId = readSelectedSessionId()
```

Comment:
:building_construction: **Crash/Merge blocker**: `sessionId` is empty when tapping a session on a tablet. Writing to SharedPreferences is too slow even when done synchronously.

Reply:
:heavy_check_mark: Fixed in `MainActivity#openSessionDetails` in 70d5f9caa29029c72b7b23d3f6faa919d5bad943. I forgot to store the `sessionId` in the `SessionDetailsFragment` case.